spec/index.bs

```diff
@@ -936,7 +933,7 @@ interface GPUValidationError {
 
 typedef (GPUOutOfMemoryError or GPUValidationError) GPUError;
 
-partial interface GPUDevice {
+partial interface mixin GPUDeviceBase {
     void pushErrorScope(GPUErrorFilter filter);
```

Comment:
Would be good to specify that the error scopes are *local* to the device reference. I.e. if two different workers start push/poping the scopes, we don't want them being mixed up arbitrarily.

Reply:
Good catch, I've updated the appropriate parts.

Reply:
Realized my previous fix was wrong, and updated again.